frontend/views/Overview.tsx

```diff
@@ -159,6 +159,7 @@ const Overview = () => {
             width: 380,
             flexShrink: 0,
             "& .MuiDrawer-paper": {
+              width: 380,
```

Comment:
```suggestion
              width: 400,
```

Reply:
in addition to that, i would allow and handle text overflow for the queue names. 
This would mean [the ListItemText component](https://github.com/PacoVK/sqs-admin/pull/622/files#diff-0edfb6ec0de6680f5bd1f012b2c7fa1c880865d25b9b2dc721a9aadc9fb09d80R220) must change to:

```
<ListItemText primary={queue.QueueName} primaryTypographyProps={{
                    style: {
                      whiteSpace: 'nowrap',
                      overflow: 'auto',
                      textOverflow: 'ellipsis'
                    }
                  }} />
```

Reply:
what do you think? if you agree and verify the fix, we could merge this :)

Reply:
![Screenshot 2023-09-09 at 17 18 31](https://github.com/PacoVK/sqs-admin/assets/112934187/652037e5-6a5e-44d9-83ad-3473716099e6)
I'm not sure, It doesn't look so user friendly

Reply:
Yeah, i see, this is what my humble frontend skills came up with first. After digging a bit more word-wrap should do the trick here. 
The ListItem can change to: 

```
<ListItemText
  primary={queue.QueueName}
  primaryTypographyProps={{
    style: {
      whiteSpace: "pre-wrap",
      overflowWrap: "break-word",
    },
  }}
/>
```

Reply:
It looks fine now. Don't worry about your frontend skills, at least you have them. 😄
I need to increase the width of the sidebar because the 'Send message' button was word-wrapping its text.